docs/DeveloperGuide.md

```diff
@@ -621,19 +628,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * 1a. User enters data in an invalid format.
 
     * 1a1. Duke The Market requests for the correct data
+
     * 1a2. User enters new data
 
-  Steps 1a1-1a2 are repeated until the data entered is correct
+    Steps 1a1-1a2 are repeated until the data entered is correct 
 
-  Use case resumes at step 2.
+    Use case resumes at step 2.
 
 * 1b. User enters data for an individual who already exists in the database.
+
     * 1b1. Duke The Market shows an error message that the user is a duplicate customer
-    * 1b2. User enters new data
 
-  Steps 1b1-1b2 are repeated until the data entered is correct.
+    * 1b2. User enters new data 
 
-  Use case resumes at step 2.
+    Steps 1b1-1b2 are repeated until the data entered is correct and not a repeat of an individual already in the database. 
+
+    Use case resumes at step 2.
 
 
 **UC03: View the overall performance of a past event**
```

Comment:
same error as above for the UC03 2a there should be an error message shown

Reply:
Same as the above I think may be incorrect to write that there's an error

Reply:
oh icic ok